docs/DeveloperGuide.adoc

```diff
@@ -798,17 +800,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *`|new user |see usage instructions |refer to instructions when I forget how to use the App
+
+|`* * *`|user |add a new person | include employees that have just entered the company
+
+|`* * *`|user |each employee has his/her dedicated timetable | trace progress of that person
```

Comment:
This particular user story doesn't adhere to the "I want to..." prefix. When read together, the whole story should be grammatically coherent.